framework/src/meshmodifiers/MeshExtruder.C

```diff
@@ -39,14 +39,23 @@ InputParameters validParams<MeshExtruder>()
 
 MeshExtruder::MeshExtruder(const InputParameters & parameters) :
     MeshModifier(parameters),
-    _num_layers(getParam<unsigned int>("num_layers")),
     _extrusion_vector(getParam<RealVectorValue>("extrusion_vector")),
+    _num_layers(getParam<unsigned int>("num_layers")),
     _map_custom_ids(false)
 {
   if (isParamValid("existing_subdomains") || isParamValid("layers") || isParamValid("new_ids"))
   {
     if (isParamValid("existing_subdomains") && isParamValid("layers") && isParamValid("new_ids"))
```

Comment:
I realize you didn't change this line but I don't believe that `"layers"` is a valid parameter at all.